content/docs/atom-guide/dev-guide/troubleshoot.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Troubleshooting Atom Renderer"
+description: "Troubleshoot GPU issues in the Atom Renderer."
+toc: true
+weight: 1000
+---
+
+This guide will help you identify and handle crashes that you might encounter in your graphics processing unit (GPU) when running **Atom Renderer**. 
+
+A key indicator of a crash in your GPU is a "device removal" failure, which occurs when Atom tries to register a Render Hardware Interface (RHI). You can find this failure in an [O3DE log file](/docs/user-guide/appendix/log-files). If you encounter this crash, follow the instructions on this page to report an issue and try several techniques to debug the problem further.
+
+
+## Reporting an issue
+
+Before reporting an issue, check for similar [existing issues](https://github.com/o3de/o3de/issues). Otherwise, [create an issue](https://github.com/o3de/o3de/issues/new/choose) and provide the following information:
+
+- Steps to reproduce the crash. Keep the information minimal and specific.
+  
+- GPU vendor and driver version (for example Nvidia, AMD, Intel, Apple, and so on).
+
+- RHI (for example Vulkan, D312, or Metal).
+
+
+## Testing alternative RHI device
```

Comment:
This section technically combines two separate concepts. Testing different GPUs, and testing different RHI backends.

Using `--forceAdapter` selects a different GPU, and using `--rhi=<rhi>` selects a different RHI backend. Users can mix and match them

Reply:
What can users input in `<rhi>`?

Reply:
"vulkan" and "dx12" works on Windows. Linux is "vulkan" only, and mac is "metal" only (so the RHI selection is only relevant on windows right now)